src/NextGen/Gateways/PayPal/PayPalStandardGateway/PayPalLogo.tsx

```diff
@@ -0,0 +1,57 @@
+export default function PayPalLogo() {
```

Comment:
Please save this as an .svg file so its cachable and doesn't add bloat.

Reply:
resolved b10b9bd3